### PR TITLE
print path of created files to stdout 

### LINKED ISSUE #189: List generated files on `--html`
Starting to use `pdoc3`, great package, fast access to documentation. 

A suggestion about README.md. The Usage section just mentions the inputs and no outputs:

> Pdoc will accept a Python module file, package directory or an import path.

It would have helped me to learn from usage section that by default pdoc3 will save generated documentation to `html/<package_name>` folder.  A user feels rather confused about not knowing what the program did.

Maybe can also add a message "Saved documentation to <path>" as console ouput.

I can do a PR, if needed.

### PR DESCRIPTION
**Pull request by epogrebnyak**
Should close #189

### CODE REVIEW

File: pdoc/cli.py [hunk 1]
```diff
@@ -322,6 +322,7 @@ def recursive_write_files(m: pdoc.Module, ext: str, **kwargs):
                 w.write(m.html(**kwargs))
             elif ext == '.md':
                 w.write(m.text(**kwargs))
+            print(f) # print created file path to stdout
```
Comment (kernc) [inline pdoc/cli.py:325]: Lint caught [some styling issues](https://travis-ci.org/github/pdoc3/pdoc/jobs/686343232#L243). Maybe prepend a blank line.

Comment (epogrebnyak) [reply pdoc/cli.py:325]: Just a sec

Comment (epogrebnyak) [reply pdoc/cli.py]: I managed to make two errors in one line: 

pdoc/cli.py:325:21: E261 at least two spaces before inline comment
pdoc/cli.py:325:57: W291 trailing whitespace

Fixed below.

**Comment by kernc** [comment]
Thank you. :cake:

**Pull request merged**